### DISCUSSION TITLE
Bad config: unknown variant `version-control`

### DISCUSSION THREAD

**Discussion by nadropper** [helix-editor/helix#6851] [Q&A] [answered]
### Summary

Helix fails to recognize `version-control` as a valid statusline element. According to the [documentation](https://docs.helix-editor.com/configuration.html), the `version-control` option is _"The current branch name or detached commit hash of the opened workspace"_.

Here's my `config.toml` file:
<img width="1680" alt="Screenshot 2023-04-22 at 12 42 15 PM" src="https://user-images.githubusercontent.com/49292621/233799154-1d67a6bb-52c5-4d6e-a143-cea95fff2ff9.png">

Here's the output when attempting to run Helix:
<img width="1680" alt="Screenshot 2023-04-22 at 12 42 39 PM" src="https://user-images.githubusercontent.com/49292621/233799164-8f7684b6-9a68-497c-b45a-d6bc5cb6b76f.png">

Here's the documentation indicating `version-control` as a statusline element.
<img width="1680" alt="Screenshot 2023-04-22 at 12 43 15 PM" src="https://user-images.githubusercontent.com/49292621/233799187-6093012f-2eb7-478a-8c61-ec04822b505a.png">

### Reproduction Steps

 

I tried this:

1. Add `version-control` as a statusline element in your `config.toml` file.
2. Run Helix.

I expected this to happen:
The current branch name or detached commit hash of the opened workspace is shown where specified.

Instead, this happened:
Helix throws the error below:
```bash
Bad config: unknown variant `version-control`, expected one of `mode`, `spinner`, `file-name`, `file-encoding`, `file-line-ending`, `file-type`, `diagnostics`, `workspace-diagnostics`, `selections`, `primary-selection-length`, `position`, `separator`, `position-percentage`, `total-line-numbers`, `spacer` for key `editor.statusline.left` at line 22 column 1
Press <ENTER> to continue with default config
```

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

zsh

### Helix Version

helix 22.12 (96ff64a8)

**Comment by pascalkuthe** [comment, marked as answer]
This feature was added in helix 23.3 but you are using helix 22.12

**Reply by nadropper** [reply]
Do you mean `23.03` (the latest release)? Because I've upgraded it via Homebrew and with this version (helix 23.03 (3cf03723)) I'm still getting the same error.

<img width="1680" alt="Screenshot 2023-04-23 at 4 55 06 PM" src="https://user-images.githubusercontent.com/49292621/233868290-12cb19a6-7361-40ac-88d7-9734380e279f.png">

**Reply by kirawi** [reply]
You typed `control-version` instead of `version-control`

**Reply by nadropper** [reply]
My mistake. It works fine! Thanks.
